api/v1/schemas.py

```diff
@@ -0,0 +1,68 @@
+from typing import Any, Dict, List, Literal, Optional
+
+from pydantic import BaseModel, Extra
+
+from .. import utils
+
+
+class Presentation(BaseModel):
+    titlePublic: Optional[str] = None
+    titleVariant: Optional[str] = None
+    attribution: Optional[str] = None
+    attributionShort: Optional[str] = None
+
+    class Config:
+        extra = Extra.forbid
+
+    def to_meta_dict(self) -> Dict[str, Any]:
+        d = {
+            "title_public": self.titlePublic,
+            "title_variant": self.titleVariant,
+            "attribution_short": self.attributionShort,
+        }
+        return utils.prune_none(d)
+
+
+class Indicator(BaseModel):
```

Comment:
Since we can mean a bunch of things by this, it's good to clarify in a comment what this represents. E.g. the JSON metadata served by the data API, or the MySQL model used by `owid-grapher`.